lib/sidekiq-scheduler/scheduler.rb

```diff
@@ -267,7 +267,9 @@ def new_job(name, interval_type, config, schedule, options)
         if job_enabled?(name)
           conf = SidekiqScheduler::Utils.sanitize_job_config(config)
 
-          if job.is_a?(Rufus::Scheduler::CronJob)
+          # `SidekiqScheduler::Utils.calc_cron_run_time` only works with a job running periodically.
+          # Also, `SidekiqScheduler::Utils.calc_cron_run_time` is only required jobs are execute frequently. So only call it if really needed.
+          if job.is_a?(Rufus::Scheduler::CronJob) && job.cron_line.rough_frequency <= 60
```

Comment:
I do not have a strong basis for this figure, but I thought 60 was sufficient, since I think this problem only occurs in cases where a batch is run every minute.

Reply:
One more note: `SidekiqScheduler::Utils.calc_cron_run_time` returns a time that zone is converted to `utc`. To keep the time zone, I think it's good to avoid calling it if possible.